gsd/test/test_fl.py

```diff
@@ -961,3 +961,61 @@ def test_read_write(tmp_path, mode):
             data10 = f.read_chunk(frame=i, name='data10')
             assert data1[0] == i
             assert data10[0] == i * 10
+
+
+@pytest.mark.parametrize('n_flush', [0, 1, 2])
+def test_flush(tmp_path, open_mode, n_flush):
```

Comment:
Would it make sense to check that after `end_frame` the chunk does not exist in the file yet? Not sure how to immediately do that since calling `read_chunk` will flush.

Reply:
Yes, we can do this by opening the file with a 2nd handle. Will add a test.